index.d.ts

```diff
@@ -0,0 +1,153 @@
+/// <reference types="node" />
+import { EventEmitter } from 'events'
+
+type Encoding = BufferEncoding | 'buffer' | null
+
+interface Writable extends EventEmitter {
```

Comment:
Q: Any reason this doesn't reference the Stream type instead since Minipass extends the Stream type?

Reply:
I was thinking that, but the definitely-typed version extends EventEmitter for some reason?

Reply:
Yeah, it seems to work fine extending from Stream (which is just an EventEmitter + `pipe(dest, options?)` after all), so I went that direction with it.